Project/DAQ_System/Program/Src/Platform/CAN/Interfaces/ican.hpp

```diff
@@ -12,12 +12,35 @@
 #ifndef ICAN_H
 #define ICAN_H
 
+// Standard Libraries
+#include <cstdint>
+#include <vector>
+
 namespace platform {
 
 class ICan {
-    public:
-        virtual ~ICan() = default;
+public:
```

Comment:
Indent this line

Reply:
I am indifferent when it comes to indenting `public:` and `private` lines. CubeIDE seems to revert it to that way, so I leave it alone.